kustomize/base/quay.deployment.yaml

```diff
@@ -33,6 +33,18 @@ spec:
                   name: cluster-trusted-ca
               - secret:
                   name: extra-ca-certs
+        - name: postgres-certs
```

Comment:
All of the changes made to this file will also need to be made to the [upgrade job](https://github.com/quay/quay-operator/blob/master/kustomize/components/job/quay.upgrade.job.yaml)

Reply:
Thanks @bcaton85 ++ requested a review to ensure I didn't messed it up again :)